setup.py

```diff
@@ -186,6 +186,17 @@ def lib_ext_kwargs(pc, prefix_env_var, lib_name, lib_pkg_name, pc_version, lib_s
         dict(extra_compile_args=cflags),
     )
 
+    if sys.platform == "linux":
+        linux_ext_kwargs = members_appended(
+            dict(sources=[platform_linux_source]),
+            lib_ext_kwargs(pc, "BORG_ACL_PREFIX", "acl", "libacl", ">=2.2.53"),
```

Comment:
I guess we need to add BORG_ACL_PREFIX to the docs.

Reply:
https://borgbackup.readthedocs.io/en/master/usage/general.html#environment-variables

Also, maybe rename it to `BORG_LIBACL_PREFIX` for better consistency.

Reply:
> I guess we need to add BORG_ACL_PREFIX to the docs.

Do you mean the man page? If so, the descriptions of other variables refer to "bundled implementation". What does it mean? I can't see any bundled libraries.

Reply:
I gave a link, just click on it, then scroll down until you find `BORG_LIBLZ4_PREFIX` - the new env var could be added right above that place, see the corresponding `general.rst` file.

Reply:
This file contains exactly the same paragraphs as `docs/man/borg.1` I mentioned (I'll update both files). Are the references to "bundeld libraries" relevant?

Reply:
Please only update the rst file, everything else will be automatically updated.